plasmapy/physics/transport.py

```diff
@@ -117,98 +177,905 @@ def Coulomb_logarithm(T, n_e, particles, V=None):
+
+    Parameters
+    ----------
+
+    T : Quantity
+        Temperature in units of temperature or energy per particle,
+        which is assumed to be equal for both the test particle and
+        the target particle
+
+    n_e : Quantity
+        The electron density in units convertible to per cubic meter.
+
+    particles : tuple
+        A tuple containing string representations of the test particle
+        (listed first) and the target particle (listed second)
+        
+    z_mean : Quantity, optional
+        The average ionization (arithmetic mean) for a plasma where the
+        a macroscopic description is valid. This is used to recover the
+        average ion density (given the average ionization and electron
+        density) for calculating the ion sphere radius for non-classical
+        impact parameters.
+
+    V : Quantity, optional
+        The relative velocity between particles.  If not provided,
+        thermal velocity is assumed: :math:`\mu V^2 \sim 3 k_B T`
+        where `mu` is the reduced mass.
+
+    Returns
+    -------
+    lnLambda : float or numpy.ndarray
+        An estimate of the Coulomb logarithm that is accurate to
+        roughly its reciprocal.
+
+    Raises
+    ------
+    ValueError
+        If the mass or charge of either particle cannot be found, or
+        any of the inputs contain incorrect values.
+
+    UnitConversionError
+        If the units on any of the inputs are incorrect
+
+    UserWarning
+        If the inputted velocity is greater than 80% of the speed of
+        light.
+
+    TypeError
+        If the n_e, T, or V are not Quantities.
+
+    Notes
+    -----
+    The Coulomb logarithm is given by
+
+    .. math::
+        \ln{\Lambda} \equiv \ln\left( \frac{b_{max}}{b_{min}} \right)
+
+    where :math:`b_{min}` and :math:`b_{max}` are the inner and outer
+    impact parameters for Coulomb collisions.
+
+    Examples
+    --------
+    >>> from astropy import units as u
+    >>> n = 1e19*units.m**-3
+    >>> T = 1e6*units.K
+    >>> particles = ('e', 'p')
+    >>> Coulomb_logarithm(T, n, particles)
+    14.748259780491056
+    >>> Coulomb_logarithm(T, n, particles, 1e6*units.m/units.s)
+    11.363478214139432
+
+    References
+    ----------
+    
+    """
+    # boiler plate checks
+    T, masses, charges, reduced_mass, V = _boilerPlate(T=T,
+                                                       particles=particles,
+                                                       V=V)
+    # collisional frequency
+    freq = collision_frequency(T=T,
+                               n_e=n_e,
+                               particles=particles,
+                               z_mean,
+                               V=V)
+    # mean free path length
+    mfp = V / freq
+    return mfp.to(units.m)
+
+
+@check_quantity({"T": {"units": units.K, "can_be_negative": False},
+                 "n": {"units": units.m**-3}
+                 })
+def Spitzer_resistivity(T,
```

Comment:
Some overlap with #199 here, I think? I recall seeing that calculated there.

Reply:
Yup there is a `resistivity` method from #199 which has some overlap with my version in the classical Spitzer case. But each of our functions has other additional methods, mine can take non-classical Coulomb logarithms while @samurai688 's version can Braginskii and ji-held resistivities.
I'm not sure what the best way to unify these two functions would be...

Reply:
I'll try to take a look at a possible solution for this :)